site/index.ejs

```diff
@@ -47,6 +47,11 @@ ga('send', 'pageview');
     <div class="js-app">
       <% if (typeof bodyContent !== 'undefined') { %><%= bodyContent %><% } %>
     </div>
+    <% if (typeof state !== 'undefined') { %>
+      <script type="text/javascript">
```

Comment:
We might need to escape the state blob here considering squarespace could send raw html, can recommend [ent](https://www.npmjs.com/package/ent).
```
import decode from 'ent/decode';
import encode from 'ent/encode';

encode(JSON.stringify(state)); // server/compile
JSON.parse(decode(state)); // client
```